inst/fw_templ/p_example/ui_body.R

```diff
@@ -60,39 +60,33 @@ body4 <- box( id = "bodyElement4",
 
 plot2_hover <- hoverOpts(id = "examplePlot2_hover")
 
-body5 <- box( id = "bodyElement5",
+body5 <- shinydashboard::box( id = "bodyElement5",
               title = "downloadablePlots - ggplot2 & lattice",
               width = 8,
               status = "info",
               collapsible = TRUE,
               collapsed = FALSE,
-              tags$table(
-                   width = "100%",
-                   tags$tr(
-                       valign = "top",
-                       tags$td(width = "45%",
-                               downloadablePlotUI("examplePlot2",
-                                                  list("jpeg", "csv"),
-                                                  "Download plot or data",
-                                                  btn_halign = "left",
-                                                  btn_valign = "top",
-                                                  btn_overlap = FALSE,
-                                                  hoverOpts = plot2_hover)),
-                       tags$td(width = "10%", HTML("&nbsp;")),
-                       tags$td(width = "45%",
-                               downloadablePlotUI("examplePlot3",
-                                                  list("png", "tiff",
-                                                       "txt", "tsv"),
-                                                  btn_overlap = FALSE,
-                                                  "Download plot or data"))) ),
+              fluidRow(
+                  column(width = 6, downloadablePlotUI("examplePlot2",
+                                                      list("jpeg", "csv"),
+                                                      "Download plot or data",
+                                                      btn_halign = "left",
+                                                      btn_valign = "top",
+                                                      btn_overlap = FALSE,
+                                                      hoverOpts = plot2_hover)),
+                  column(width = 6, downloadablePlotUI("examplePlot3",
+                                                      list("png", "tiff",
+                                                           "txt", "tsv"),
+                                                      btn_overlap = FALSE,
+                                                      "Download plot or data")) ),
```

Comment:
Looks nice, but out of curiosity is there a technical reason for the change?

Reply:
when switching between hiding and showing the right sidebar, the content is resized with the fluidRow but not with the table, so it's for responsiveness. We could also add a scrollbar around the table instead?

Reply:
Nope, I think that's fine.